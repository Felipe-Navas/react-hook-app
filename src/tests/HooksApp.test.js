import React from 'react';
import '@testing-library/jest-dom';
import { shallow } from 'enzyme';
import { HooksApp } from '../HooksApp';

describe('Testing the component HooksApp', () => {
  test('should render', () => {
    const wrapper = shallow(<HooksApp />);
    expect(wrapper).toMatchSnapshot();
  });
});
