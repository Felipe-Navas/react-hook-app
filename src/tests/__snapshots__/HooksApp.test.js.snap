// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing the component HooksApp should render 1`] = `
<div>
  <h1>
    Hola Mundo
  </h1>
</div>
`;
